### PR TITLE
[JENKINS-72143] French translation

### PR DESCRIPTION
**Pull request by jgreffe**
See [JENKINS-72143](https://issues.jenkins.io/browse/JENKINS-72143).

Reported missing keys from hudson/Messages.properties to related Messages_fr.properties

### Proposed changelog entries

- Translate hudson/Messages.properties into French.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:25]:
```suggestion
  Les espaces ne peuvent plus être utilisés comme séparateurs. Merci d''utiliser à la place maintenant ’,’ comme séparateur.
```
Subjective

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:27]:
```suggestion
  ’{0}’ ne correspond à rien, mais ’{1}’ oui. Peut-être est-ce cela que vous vouliez dire?
```

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:56]:
```suggestion
  Obsolète - Veuillez utiliser le droit "Overall/Administer" à la place
```
For consistency with other translations including english permission terms.

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:58]:
```suggestion
  Obsolète - Veuillez utiliser le droit "Overall/Administer" à la place
```
Same as above.

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:59]:
```suggestion
PluginManager.PluginCycleDependenciesMonitor.DisplayName=Moniteur de dépendances cycliques
```
For consistency with L61?

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:63]:
```suggestion
PluginManager.UpdateSiteError=Erreur lors de la vérification du site de mise à jour pour {0} essai(s). La dernière exception était : {1}
```
For consistency with L62 above

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:65]:
```suggestion
PluginManager.UnexpectedException=Exception inattendue lors la vérification du serveur de mise à jour
```
Same as above.

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:76]:
```suggestion
 Avertissement : ce plugin n''est compatible que pour Jenkins {0} ou plus récent. \
```

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:79]:
```suggestion
 Avertissement : ce plugin nécessite des versions plus récentes de dépendances et au moins un de ces dépendances n''est pas compatible avec la version installée. \
```
The link "dependency" = "plugin" isn't obvious, this equivalent formulation seems better to me.

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:80]:
```suggestion
 C''est habituellement le cas parce que le comportement de ce plugin de dépendance ou ses APIs ont changé, ou parce qu''il utilise un format de configuration différent de celui installé. \
```
To make the link (cf suggestion above)

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:97]:
```suggestion
  Vous utilisez une URL invalide pour télécharger le plugin. Les URLs de type "file", "https" et "http" (non recommandé) sont supportées.
```

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:100]:
```suggestion
  Le site de mise à jour est anormalement vide. Veuillez entrer une URL valide.
```
Subjective, seems more easier to understand that way.

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:127]:
```suggestion
PluginWrapper.Already.Disabled=Le plugin "{0}" était déjà désactivé
PluginWrapper.Plugin.Has.Dependent=Le plugin "{0}" a au moins un plugin dépendant ({1}) et la stratégie de désactivation est {2} ainsi, il ne peut pas être désactivé
PluginWrapper.Plugin.Disabled=Plugin "{0}" désactivé
PluginWrapper.NoSuchPlugin=Aucun plugin trouvé avec le nom "{0}"
PluginWrapper.Error.Disabling=Une erreur a été relevée lors de la désactivation du plugin "{0}". Erreur : "{1}"
```
Use of the double quotes character instead of doubling the single quote character.

**Review by lemeurherve — COMMENTED** [review]
Optional suggestions, good job!

**Comment by abhishekmaity** [comment]
\label localization

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (jgreffe) [inline core/src/main/resources/hudson/Messages_fr.properties:79]:
```suggestion
 Avertissement : ce plugin nécessite des versions plus récentes de dépendances et au moins une de ces dépendances n''est pas compatible avec la version installée. \
```

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (lemeurherve) [inline core/src/main/resources/hudson/Messages_fr.properties:88]:
```suggestion
  Avertissement : cette version de plugin n''est peut-être pas sûre. Veuillez vérifier les notices de sécurité suivantes :
```

**Review by lemeurherve — COMMENTED** [review]
Last suggestion fixing a typo, LGTM otherwise!

File: core/src/main/resources/hudson/Messages_fr.properties
Comment (jgreffe) [reply core/src/main/resources/hudson/Messages_fr.properties]: Handled here: https://github.com/jenkinsci/jenkins/pull/8578/commits/94b7c54730f0eb3e3e9f5bce765511f540b9a488#diff-e77cc092b639372715fe10bdee539f1e86bedb251cc4c34e083d60f2703dae1aR79

**Review by lemeurherve — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
